cfgmgr/buffermgrdyn.cpp

```diff
@@ -1462,6 +1469,19 @@ task_process_status BufferMgrDynamic::refreshPgsForPort(const string &port, cons
                 continue;
             }
 
+            if (cable_length == "0m" && it->second.lossless)
```

Comment:
let's use the reference to it->second defined in L1459.
```suggestion
            if (cable_length == "0m" && portPg.lossless)
```